src/elements/element.line.js

```diff
@@ -145,35 +326,106 @@ class Line extends Element {
 		super(props);
 	}
 
-	draw(ctx) {
+	set points(points) {
+		this._children = points;
+		delete this._segments;
+	}
+
+	get points() {
+		return this._children;
+	}
+
+	get segments() {
+		return this._segments || (this._segments = computeSegments(this));
+	}
+
+	first() {
+		const segments = this.segments;
+		const points = this.points;
+		return segments.length && points[segments[0].start]._view;
+	}
+
+	last() {
+		const segments = this.segments;
+		const points = this.points;
+		const count = segments.length;
+		return count && points[segments[count - 1].end]._view;
+	}
+
+	interpolate(point, axis) {
```

Comment:
Do we want to make any of the new methods in this class private?

Reply:
I'm undecided on that

Reply:
Those are mostly for the filler plugin, so I guess they are public in that sense.  Filler is still internal though, so could go either way. I think we will be modifying the public interface quite a lot still, before v3 is out. So I'm fine either way.